lib/auth.js

```diff
@@ -269,11 +269,29 @@ internals.Auth.prototype._authenticate = function (request, next) {
 
         // Check scope
 
+        var expandScope = function (scope) {
```

Comment:
Does this needs to be a function? Why not just inline the logic inside the for loop?